src/stubs/progressbar/__init__.pyi

```diff
@@ -19,13 +19,19 @@ from .widgets import Widgets
 
 class ProgressBar:
     max_value: int
+    value: int
 
-    def __init__(self, max_value: int, widgets: Optional[List[Union[Widgets, str]]]) -> None: ...
+    def __init__(self, max_value: Union[int, type[UnknownLength]], widgets: Optional[List[Union[Widgets, str]]]) -> None: ...
```

Comment:
Can you expand on this change?

Reply:
Pretty simple really. If you don't know how big the file is, you use UnknownLength, which is an object provided by progressbar. Then the progressbar won't show progress per se, but just how fast the download is and that it is actually downloading.